Frontend/CO.CDP.OrganisationApp/Pages/Support/OrganisationApproval.cshtml.cs

```diff
@@ -0,0 +1,47 @@
+using CO.CDP.Organisation.WebApiClient;
+using Microsoft.AspNetCore.Mvc;
+using OrganisationWebApiClient = CO.CDP.Organisation.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Pages.Support;
+
+public class OrganisationApprovalModel(
+    IOrganisationClient organisationClient,
+    ISession session) : LoggedInUserAwareModel(session)
+{
+    public OrganisationWebApiClient.Organisation? OrganisationDetails { get; set; }
+
+    [BindProperty]
+    public Boolean? Approval { get; set; }
```

Comment:
Seems like no validation, if user has not selected the option.
Ideally user must select an option before continuing with the page?

Reply:
So basically Luke has confirmed that we need to have validation, otherwise how you keep in Pending State on Continue button click?